crc_errors/CRC_Count_check.py

```diff
@@ -28,50 +25,43 @@
 class common_setup(aetest.CommonSetup):
     """ Common Setup section """
 
-    # CommonSetup have subsection.
-    # You can have 1 to as many subsection as wanted
-
     # Connect to each device in the testbed
     @aetest.subsection
     def connect(self, testbed):
-        genie_testbed = Genie.init(testbed)
+        genie_testbed = load(testbed)
         self.parent.parameters['testbed'] = genie_testbed
         device_list = []
         for device in genie_testbed.devices.values():
```

Comment:
there's a minor logic flaw here in the code
`device_list` is a list of all devices in the current testbed at any time, because on the first device we couldn't connect to, `self.failed()` will stop the script execution, and block the remaining script from running.

so it's really not useful to introduce parameter `dev`

Reply:
Not sure why Kevin introduced the ```dev``` parameter.  Might be useful from what you have pointed out further down though.